### PR TITLE
Change mount utils to "k8s.io/mount-utils"

### PR DESCRIPTION
**Pull request by jingxu97**
k8s.io/utils/mount is deprecating. Change to use k8s.io/mount-utils

**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespaces from that line:
>
> /kind api-change
> /kind bug
> /kind cleanup
> /kind design
> /kind documentation
> /kind failing-test
> /kind feature
> /kind flake

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

**Comment by jingxu97** [comment]
cc @andyzhangx

File: go.mod [hunk 1]
```diff
@@ -28,8 +27,9 @@ require (
 	k8s.io/component-base v0.19.0
 	k8s.io/klog v1.0.0
 	k8s.io/kubernetes v1.18.0
+	k8s.io/mount-utils v0.20.6
 	k8s.io/test-infra v0.0.0-20200115230622-70a5174aa78d
-	k8s.io/utils v0.0.0-20200324210504-a9aa75ae1b89
```
Comment (msau42) [inline go.mod:32]: This line should be removed.

Comment (jingxu97) [reply go.mod:32]: I see there are places like using k8s.io/utils/exec etc. 
https://github.com/kubernetes-sigs/gcp-compute-persistent-disk-csi-driver/search?q=%22k8s.io%2Futils%2Fexec%22

Comment (jingxu97) [reply go.mod:32]: So I think we should leave this line here? @msau42

Comment (msau42) [reply go.mod:32]: yup thanks for checking!

**Pull request merged**